modules/runner/src/main/ts/core/Hook.ts

```diff
@@ -1,6 +1,6 @@
 import { ExecuteFn, Hook } from '@ephox/bedrock-common';
 import * as Runnable from './Runnable';
 
-export const createHook = (title: string, fn: ExecuteFn): Hook => {
+export const createHook = (title: string, fn: ExecuteFn | undefined): Hook => {
```

Comment:
Shouldn't this be the other way around?
```suggestion
export const createHook = (title: string | undefined, fn: ExecuteFn): Hook => {
```

Reply:
No, this is the internal implementation not the public one. A title is mandatory for running and you'll see it's normalized to this in `Globals`.